modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -64,53 +70,67 @@ http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/m
 ----
 =====
 
-Because CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes, there is no need to do any other special configuration.
+CloudHub synchronizes your schedules automatically, overriding any synchronize property set on the flows containing the polling scopes, so you don't need to do any other special configuration.
 
-Once it is deployed, select the application in the Runtime Manager console, and click *Schedules* in the left navigation menu. Runtime Manager lists the polling elements in your application in the order they appear in your application:
+After deploying an app, you can view the polling elements in Runtime Manager by selecting the app and clicking *Schedules* in the left navigation menu. Runtime Manager lists the polling elements in your application in the order they appear in your application:
 
-image::schedulesview.png[SchedulesView]
+image::ch-view-polling-schedule.png[View Schedules]
 
-[NOTE] 
-The name of each element is determined by the `doc:name` attribute in the polling scopes, if you add one in the XML. If you do not specify a `doc:name`, then CloudHub applies the name, *yourFlowname Poll*.
+The name of each element is determined by the `doc:name` attribute in the polling scopes, if specified in the XML. If you do not specify a `doc:name`, CloudHub applies the name, *yourFlowname Poll*.
 
-The *Last Run* column shows the exact date and time of each recently run polling element. You can view the Logs to see further details about when the jobs started and ended their scheduled runs.
+The *Last Run* column shows the date and time each polling element last ran. You can view the logs to see more information about when the jobs started and ended scheduled runs.
 
 == Manage Schedules at Runtime
 
-To manage the polling elements:
+To manage schedules in Runtime Manager:
 
-. Check the boxes next to the individual elements you want to manage.
-. Click *Run now* if you want to override the schedule and run the job immediately.
-. Click *Disable* to suspend a polling element. CloudHub does not run the polling job until you *Enable* it once again.
-. In the *Schedule* column, CloudHub lists each polling element's polling frequency.
-. Click the frequency of a polling element to open the schedule editor:
+. Click *Applications* in the left pane, click the app, and then click *Schedules*:
 +
-image::scheduleschangetime.png[SchedulesChangeTime]
+image::ch-schedule-change.png[View Schedules]
 +
-. In the schedule editor, adjust the polling frequency of the element, or disable the polling frequency by unchecking *Enabled*:
+. Select the elements to manage.
+
+* Click *Run now* to override the schedule and run the job immediately.
```

Comment:
```suggestion
. Click *Run now* to override the schedule and run the job immediately.
```

Reply:
Here and below, STET * if these are choices. If they are chronological, they should be .